classes/controller/AdminController.php

```diff
@@ -4166,8 +4166,21 @@ protected function processBulkStatusSelection($status)
                 /** @var ObjectModel $object */
                 $object = new $this->className((int) $id);
                 $object->active = (int) $status;
-                $result &= $object->update();
+                $update_ok = $object->update();
+                $result &= $update_ok;
+
+                if (!$update_ok) {
+                    $this->errors[] = $this->trans('Can\'t update #%id% status', ['%id%' => (int) $id], 'Admin.Notifications.Error');
+                }
             }
+
+            if ($result) {
+                $this->redirect_after = self::$currentIndex . '&conf=5&token=' . $this->token;
+            }
+
+            $this->errors[] = $this->trans('An error occurred while updating the status.', [], 'Admin.Notifications.Error');
```

Comment:
If it's good, we have this error ?

Reply:
Hi @Progi1984  I found it here : https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Controller/Admin/Improve/International/CurrencyController.php#L642

Ping @LouiseBonnard

Reply:
Yep, it exists!

Reply:
Thank you @LouiseBonnard  ! you can validate the wording now please

Reply:
@Julievrz is now in charge of the wording reviews, she will check it as soon as she can. :-)

Reply:
Oh Sorry @LouiseBonnard

Reply:
No. I talk about the behavior ! I would add error in `else` for the `if $result`.

Reply:
Well seen! Thank you @Progi1984